operator/src/main/java/org/bf2/operator/managers/MetricsManager.java

```diff
@@ -115,7 +117,8 @@ private void createOrUpdateMetrics(Kafka kafka) {
 
     public static Tags buildKafkaInstanceTags(HasMetadata obj) {
         ObjectMeta metadata = obj.getMetadata();
-        return Tags.of(Tag.of(TAG_LABEL_NAMESPACE, metadata.getNamespace()), Tag.of(TAG_LABEL_INSTANCE_NAME, metadata.getName()), OWNER);
+        String profileType = OperandUtils.getOrDefault(metadata.getLabels(), ManagedKafka.PROFILE_TYPE, "standard");
```

Comment:
I'm +1 to adding the profile type to the labels of the metrics emitted by fleetshard.  I see that as being generally useful.

I am wondering about the label name.  Should it just be `profile_type` or `instance_profile_type` for consistency with the other label names already on the metric emitted by fleetshard?

Reply:
@k-wall , the naming also caught my eye here. I think shortening to `profile_type` would be good as well.

Reply:
One thing to consider is the ergonomics of using it in queries that already have the label set as `bf2_org_kafkaInstanceProfileType` — if we name it something else then queries become a little more complex/cumbersome.

I'm fine with it either way though :slightly_smiling_face:

Reply:
I think the label is set on Kafka metrics as ` bf2_org_kafkaInstanceProfileType` but not on fleetshard metrics.